app/commands/v2/unpublish.rb

```diff
@@ -132,7 +133,23 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
+        scope = filter.filter(locale: locale, state: allowed_states)
+
+        Queries::GetLatest.call(scope).first
+      end
+
+      def supersede_previous_unpublished_items(content_item)
+        items = ContentItemFilter.similar_to(
+          content_item,
+          state: "unpublished",
+          base_path: nil,
+          user_version: nil,
+        )
+
+        items.each do |item|
+          next if item == content_item
+          State.supersede(item)
+        end
```

Comment:
(have talked with @tuzz and he was already on top of the changes to the publish endpoint...)